frappe/desk/doctype/todo/todo_calendar.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
+// License: GNU General Public License v3. See license.txt
+
+frappe.views.calendar["ToDo"] = {
+	field_map: {
+		"start": "date",
+		"end": "date",
+		"id": "name",
+		"title": "owner",
```

Comment:
@vishdha the content of the calendar event should have the title of ToDo instead of the owner. Because users other than System Managers can view only their own ToDos.